templates/Java/libraries/okhttp-gson/JSON.mustache

```diff
@@ -0,0 +1,548 @@
+{{>licenseInfo}}
```

Comment:
Which lines in this file are different than the base template? Can you provide information about why this was included and what it supports in your solution?

Reply:
108, 120, 130,134 - gson requires to register some class to serialize/deserialize in the proper way.